src/network.c

```diff
@@ -685,7 +685,7 @@ size_t readn_ssl(SSL *ssl, void *buf, size_t amount) {
  * @param fd The file descriptor to read from
  * @param buf The buffer to read into
  * @param amount The the amount of data that can be read into buf
- * @return The number of bytes read
+ * @return The number of bytes read, or a negative error code.
```

Comment:
We normalized the return values for the write* functions in https://github.com/m-lab/ndt/pull/31

This may be an opportunity to help normalize the read* functions so that readn_raw is more concise.